foxpuppet/browser/windows.py

```diff
@@ -2,6 +2,8 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from selenium.webdriver.common.by import By
+
 
 class Windows(object):
```

Comment:
This looks to be a general windows class not related to the browser only. So it should be kept outside of the browser folder.